Src/VimCore/KeyInput.fs

```diff
@@ -81,6 +81,17 @@ type KeyInput
     interface System.IComparable<KeyInput> with
         member x.CompareTo other = x.CompareTo other
 
+[<Sealed>]
+type KeyInputData
+    (
+        _keyInput: KeyInput,
+        _wasMapped: bool
+    ) =
+
+    member x.KeyInput = _keyInput
+    member x.WasMapped = _wasMapped
+    static member Create keyInput wasMapped = KeyInputData(keyInput, wasMapped)

```

Comment:
Consider adding a `ToString` override to facilitate debugging

Reply:
Great idea. Done.

![image](https://user-images.githubusercontent.com/2680524/58453080-be4c8f80-80e7-11e9-9830-6bcd33109d12.png)